test/integration/package_int_test.go

```diff
@@ -755,6 +755,37 @@ func (suite *PackageIntegrationTestSuite) TestCVE_Indirect() {
 	cp.ExpectExitCode(1)
 }
 
+func (suite *PackageIntegrationTestSuite) TestChangeSummary() {
+	suite.OnlyRunForTags(tagsuite.Package)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("config", "set", constants.AsyncRuntimeConfig, "true")
+	cp.Expect("Successfully set")
+	cp.ExpectExitCode(0)
+
+	cp = ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/small-python", "."),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
+	)
+	cp.Expect("Checked out", e2e.RuntimeSourcingTimeoutOpt)
+	cp.ExpectExitCode(0)
+
+	cp = ts.SpawnWithOpts(
+		e2e.OptArgs("install", "requests@2.31.0"),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
```

Comment:
Asy.. you get it :p

Reply:
I don't follow, this one is specifically enabling runtimes?

Reply:
This test needs to enable runtimes, or else the code in the linked conditional will not fire, and you will not get any dependency information or CVE information.

Reply:
Oh I see what you mean. So this IS using Async runtimes, but because our integration tests always disable runtime sourcing and it's instead an opt-in, we have to opt-in here because otherwise the CVE info isn't printed. 

I think that's fine for now. We have a story for combining async runtimes with the disabled runtime mechanic, in so far as that's reasonable. Additionally we have a story about having minimal runtimes. I suspect soon the disabled runtime functionality will be gone altogether.